server/db/mysql/adapter.go

```diff
@@ -29,6 +30,10 @@ type adapter struct {
 	// Maximum number of message records to return
 	maxMessageResults int
 	version           int
+
+	// Single query and transaction timeouts (in seconds).
```

Comment:
Go tools like IDEs show comments in tool tips when the mouse hovers over a variable. It's a good practice to comment each variable or function individually.

Reply:
Done.